samples/azure-service-bus-netstandard/send-reply/ASBS_1/Endpoint1/Program.cs

```diff
@@ -19,7 +19,7 @@ static async Task Main()
         var connectionString = Environment.GetEnvironmentVariable("AzureServiceBus_ConnectionString");
         if (string.IsNullOrWhiteSpace(connectionString))
         {
-            throw new Exception("Could not read the 'AzureServiceBus.ConnectionString' environment variable. Check the sample prerequisites.");
+            throw new Exception("Could not read the 'AzureServiceBus_ConnectionString' environment variable. Check the sample prerequisites.");
```

Comment:
Should this be fixed in the sample code as well?

Reply:
the snippet already comes from the sample, so we're good.